app/policies/rubygem_policy.rb

```diff
@@ -39,22 +39,26 @@ def request_ownership?
   end
 
   def close_ownership_requests?
-    rubygem_owned_by?(user)
+    rubygem_owned_by?(user, minimum_required_role: Access::OWNER)
```

Comment:
Since we're usually doing the same thing a bunch, you could make `rubygem_owned_by?(user)` always require ownership level access.
Then you add a new one for `rubygem_maintained_by?(user)`

Reply:
I've separated out the behavior that checks for just an ownership and ownership + role/permission into separate methods.

Reply:
I've left the original `owned_by?` method alone and moved the role checking to `owned_by_with_role?`